google/cloud/security/inventory2/resources.py

```diff
@@ -383,6 +399,24 @@ def iter(self):
                 yield FACTORIES['firewall'].create_new(data)
 
 
+class InstanceGroupIterator(ResourceIterator):
+    def iter(self):
+        gcp = self.client
+        if self.resource.enumerable():
+            for data in gcp.iter_computeinstancegroups(
+                projectid=self.resource['projectId']):
+                yield FACTORIES['instancegroup'].create_new(data)
+
+
+class BackendServiceIterator(ResourceIterator):
+    def iter(self):
+        gcp = self.client
+        if self.resource.enumerable():
+            for data in gcp.iter_backendservices(
+                projectid=self.resource['projectId']):
```

Comment:
Indentation will probably need to be fixed to make linter happy.
For control structures like if/for etc, when you wrap to a new line indent twice so it becomes obvious where the body starts since the body will be indented once.